src/main/kotlin/racingcar/Moveable.kt

```diff
@@ -10,12 +8,11 @@ interface Moveable {
     }
 
     class Random : Moveable {
-        private val intRandom = IntRandom.Smart(ThreadLocalRandom.current(), BOUND_MAX)
+        private val intRandom = IntRandom.Smart(0 until 10)
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        private val intRandom = IntRandom.Smart(0..9)
```

Reply:
요구사항이 0~9 라고 되어 있는데 왜 10이라고 생각했을까요. ㅎㅎ
ad94d7a7d0360f0f1d6c859c39c9f01979ec4861 에서 반영하였습니다.